core/fifo_cache.go

```diff
@@ -44,7 +44,7 @@ func (f *BufferFIFOCache[K, V]) Put(key K, val V) {
 	f.l.Lock()
 	defer f.l.Unlock()
 
-	f.buffer.Insert(key) // Insert will remove the oldest [K] if we are at the [limit]
+	_ = f.buffer.Insert(key) // Insert will remove the oldest [K] if we are at the [limit]
```

Comment:
Can we comment that `remove` always returns `nil`, so it is safe to ignore this error here.